api/src/apollo/User/mutations.ts

```diff
@@ -0,0 +1,73 @@
+import { idArg, inputObjectType, mutationField, nonNull } from 'nexus';
+
+export const UserCreateInputType = inputObjectType({
+  name: 'UserCreateInputType',
+  definition(t) {
+    t.nonNull.string('email');
+    t.nonNull.string('uid');
+    t.string('name');
+    t.string('photo');
+    t.gender('gender');
+    t.date('dob');
+    t.string('state');
+    t.string('city');
+    t.string('college');
+    t.string('stream');
+    t.string('mobile');
+    t.string('selfId');
+    t.list.string('festId');
```

Comment:
This also can't be input. Will be updated later on payment / webmail verification